ChangeLog.d/mbedtls_ssl_hs_cb_t.txt

```diff
@@ -0,0 +1,3 @@
+Features
+   * Introduce mbedtls_ssl_hs_cb_t typedef.
+   * Inline func for mbedtls_ssl_conf_cert_cb().
```

Comment:
I don't think this is major enough to warrant an entry in the changelog.

Reply:
:rocket: